types/index.d.ts

```diff
@@ -0,0 +1,2 @@
+export function setCredentials(id: any, key: any): boolean;
```

Comment:
```suggestion
export function setCredentials(id: string, key: string): boolean;
```

Reply:
Now that I think of it, why does the setCredentials function need to return anything at all?

Reply:
Good catch it should not - updating it to be void